broker/src/main/java/io/camunda/zeebe/broker/clustering/topology/ClusterTopology.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.clustering.topology;
```

Comment:
I'm not perfectly happy with putting all of these in broker. I feel we have to move it somewhere else. But I guess we can wait until we built the rest of the components and refactor later.